index.html

```diff
@@ -10,188 +10,223 @@
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"/>
 </head>
 <body>
-    <header>
-        <a href="#" class="mylogo">Miftah</a>
-        <nav class="mob-version">
-           <i class="fa fa-bars baricon">
-           </i>
+    <header class="header-section" >
+        <h1 class="logo">Miftah</h1>
+        <nav class="nav-items">
+            <div class="mob-bar">
+                <i class="fa fa-bars baricon"></i>
+            </div>
+            <ul class="nav-list">
+                <li>Portoflio</li>
+                <li>About</li>
+                <li>Contact</li>
+                <li><i class="fa fa-envelope"></i></li>
+              </ul>
         </nav>
     </header>
     <!-- Headlines -->
     <section class="content">
         <div class="header-text">
-            <h1 class="title-1">Miftah Amin</h1>
+            <h2 class="title-1">Miftah Amin</h2>
             <h2 class="title-2">Software developer</h2>
-            <p class="para">
-                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-            </p>
+            
         </div>
     </section>
-    <hr class="hrbar">
-    <div class="technologies">
-        <ul class="tec-section">
-            <li class="tec lang">
-                <h2>Langguages</h2>
-                <ul class="featureslist"> 
-                <li>Javascript</li>
-                <li>Ruby</li>
-                <li>Html</li>
-                <li>Css</li>
-                </ul>
-            </li>
-            <li class="tec farme ">
-                <h2>Frameworks</h2>
-                <ul class="featureslist"> 
-                <li>Bootstrap</li>
-                <li>Ruby on Rails</li>
-                <li>RSPec</li>
-                <li>CapyBara</li>
-                <li>Selenium</li>
-                </ul>
-            </li>
-            <li class="tec skills">
-                <h2>Skills</h2>
-                <ul class="featureslist"> 
-                <li>Codekit</li>
-                <li>Github</li>
-                <li>Codepen</li>
-                <li>Gitlab</li>
-                <li>Terminal</li>
-                </ul>
-            </li>
-          </ul>
-    </div>
+    <section class="desc-section">
+            
+        <p class="para">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        
+        <div class="technologies">
+            <ul class="tec-section">
+                <li class="tec lang">
+                    <h2>Langguages</h2>
```

Comment:
- [ ] Let's correct this typo error here from `Langguages` to `Languages` 💯 so that your page looks professional 👍

Reply:
Thank you, changed to Languages Languages 👍